source/calculus/source/meta/frontmatter.ptx

```diff
@@ -67,12 +67,14 @@
       </author>
     </credit>
     <date><today /></date>
-  </titlepage>
+    <xi:include href="../../../common/website.ptx"/>
+    <xi:include href="../../../common/copyright.ptx" />
+  </bibinfo>
+  <titlepage><titlepage-items/></titlepage>
+
 
 
   <colophon xml:id="colophon">
-    <xi:include href="../../../common/website.ptx"/>
-    <xi:include href="../../../common/copyright.ptx" />
     <p>
```

Comment:
I'm not seeing where this is showing up in the Preview anymore

Reply:
Ugh, this was supposed to be attached to the subsequent lines where we acknowledge inclusion of other CC licensed stuff, and NSF support

Reply:
The NSF support is included on the TBIL Resource Library page now, so I think nsf.ptx can actually be deprecated.  But we do need to acknowledge the CC licensed stuff we use